src/main/java/de/zalando/aruha/nakadi/validation/EventBodyMustRespectSchema.java

```diff
@@ -31,8 +31,10 @@
     @Override
     public EventValidator materialize(final EventType eventType, final ValidationStrategyConfiguration vsc) {
 
-        final JSONSchemaValidator defaultSchemaValidator = new JSONSchemaValidator(
-                new JSONObject(eventType.getSchema().getSchema()));
+        JSONObject effectiveSchema = EventValidation.effectiveSchema(eventType);
```

Comment:
Maybe, we should check, in the case of a business event, that the schema provided by the user doesn't contain a metadata field.